commit 6fa1f6f15284a643a7aafb76093d6b47102e5df1
Author: Trond Myklebust <trond.myklebust@primarydata.com>
Date:   Fri Feb 17 19:49:09 2017 -0500

    pNFS/flexfiles: If the layout is invalid, it must be updated before retrying
    
    commit df3ab232e462bce20710596d697ade6b72497694 upstream.
    
    If we see that our pNFS READ/WRITE/COMMIT operation failed, but we
    also see that our layout segment is no longer valid, then we need to
    get a new layout segment before retrying.
    
    Fixes: 90816d1ddacf ("NFSv4.1/flexfiles: Don't mark the entire deviceid...")
    Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
    Signed-off-by: Anna Schumaker <Anna.Schumaker@Netapp.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1335_6fa1f6f15284a643a7aafb76093d6b47102e5df1.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1335_6fa1f6f15284a643a7aafb76093d6b47102e5df1.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
